### DISCUSSION TITLE
Imperfect walls after seam

### DISCUSSION THREAD

**Discussion by gserra07** [OrcaSlicer/OrcaSlicer#5130] [Q&A]
Hi.
I have always used cura and I am starting to use the orca slicer and I am liking it a lot more, but I have a problem with certain pieces. I have noticed that it happens in the sewing area just when starting the perimeter and only on the external perimeter.
I have tried lowering accelerations and speeds but it still happens (less than before).
I have lowered accelerations and speeds to leave them the same as they were in the cure (it doesn't happen to me there) and I don't know if any other parameter that comes preconfigured or something like that could be affected.
My printer is an artillery genius.
Thanks in advance
![image](https://github.com/SoftFever/OrcaSlicer/assets/10289311/c6c76fd8-d530-4996-bcdd-429fa8480ba7)

**Comment by kkunk** [comment]
looks like pressure advance is too high

**Reply by gserra07** [reply]
Thanks for your answer.
I have done different tests modifying the pressure advance. I have tried with 0.3 and -0.2 and they always come out the same
![image](https://github.com/SoftFever/OrcaSlicer/assets/10289311/8b156375-736f-4ae9-ba35-44c460771b77)
